controllers/nginx/Makefile

```diff
@@ -3,7 +3,7 @@ all: push
 BUILDTAGS=
 
 # 0.0 shouldn't clobber any release builds
```

Comment:
No I'm saying 0.0 is unused, or if anything is there it isn't a release build so you can use that tag for testing